dis_snek/utils/attr_utils.py

```diff
@@ -25,8 +25,7 @@
 _T = TypeVar("_T")
 
 
-def define(cls: _T) -> _T:
-    return attr.define(cls, **class_defaults)
+define: Callable[[_T], _T] = partial(attr.define, **class_defaults)
```

Comment:
Why is this part of the pr? Doesnt look like its used for `wait_for_component`

Reply:
Check #general and #lib-discussion, Proxy, Astrea and I were talking about this, it's there because #62 wasn't properly tested and basically broke the lib, so we decided to revert

Reply:
Yes ik, but you shouldn't do it in this PR. It's completely not related to `wait_for_component()`

Reply:
Should be noted that #64 should *properly* fix this bug. But yeah, keep this for local testing but *not* for this PR.

Reply:
Got it, thanks!

Reply:
Reverted this and upstreamed